spec/components/sdg/goals/index_component_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+describe SDG::Goals::IndexComponent, type: :component do
+  let!(:goals) { SDG::Goal.all }
+  let!(:phases) { SDG::Phase.all }
+  let!(:component) { SDG::Goals::IndexComponent.new(goals, phases) }
+
+  before do
+    Setting["feature.sdg"] = true
+  end
+
+  it "renders a heading" do
+    render_inline component
+
+    expect(page).to have_css "h1", exact_text: "Sustainable Development Goals"
+  end
+
+  it "renders phases" do
+    render_inline component
+
+    expect(page).to have_content "Sensitization"
+    expect(page).to have_content "Planning"
```

Comment:
You could avoid 3 expect and use de `and` method.
like 
```
expect(page).to have_content("Sensitization").and(have_content("Planning")).and(have_content("Monitoring"))
```

Reply:
Thanks for the comment! However, since the "one expectation per line" pattern is used in the rest of the code, here we're using it for consistency.